### ISSUE TITLE
Extension breaks Gitlab User Preferences -> Behaviour -> Layout width

### ISSUE THREAD

**Issue by sl33kr** [tomasbonco/gitlabtree#17]
Firstly this plugin is a must have for any Gitlab user coming from Bitbucket! Amazing work.

In Gitlab 11.1.4-ee when a user has selected "Layout width" Fluid, extension causes changes to still appear limited.

Looks like in 11.1.4 the limiting CSS is applied in a higher parent. Regardless of setting the changes html still appears like:
```
<div>
    <div class="diffs tab-pane active" id="diffs">
        ...
    </div>
</div>
```

It appears this extension is adding:
```
<div class="container-fluid container-limited limit-container-width">
    <div class="diffs tab-pane active" id="diffs">
        ...
    </div>
</div>
```

Simply removing the `class="container-fluid container-limited limit-container-width"` appears to respect the Gitlab user setting.

**Comment by sl33kr** [comment]
Scratch it, just tried the master branch and can see this is all fixed!

Will wait for that version.

**Comment by sl33kr** [comment]
Sorry I should have asked, is there a timeline for the release of the refactored version?

**Comment by tomasbonco** [comment]
Hi!

First of all, thank you for your kind words. I really appreciate how much time you spent on this issue, because it’s detailed.

The code is mostly finished, I’m testing it right now. I’m aware of the situation and want to release a new version asap. I still need to update README, changelog, and stuff. So maybe today, but this week for sure.

**Comment by tomasbonco** [comment]
I just released v1.0.0. Hope you'll like it.
More about the release: https://github.com/tomasbonco/gitlabtree/releases/tag/v1.0.0
